### DISCUSSION TITLE
изменение видимости репозитория с private на public

### DISCUSSION THREAD

**Discussion by Nikolay449** [community/community#76026] [Repositories]
### Select Topic Area

Question

### Body

Не получается изменить видимость репозитория с private на public. Кнопка  Change visibility не отображается.
![2023-11-19_15-31-05](https://github.com/community/community/assets/118308231/92d3bbcb-96fa-4587-a95e-82551747d689)

**Comment by ZhengLinLei** [comment]
Can you provide an english version of your question?

I think you are asking why your repository cannot be changed to private. As you can see the text below the title `For security reasons, you cannot change the visibility of a fork`. This is because your repo is a fork from other repo.

**Reply by VladimirCreator** [reply]
I always wondered what these security reasons are. Do you have any ideas?

**Comment by queenofcorgis** [comment]
Hi @Nikolay449 , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be closing and  locking any posts in languages other than English for now, including this one.
